module_utils/network_lsr/argument_validator.py

```diff
@@ -1911,3 +1973,26 @@ def _ipv4_enabled(connection):
                     "Setting DNS options 'inet6', 'ip6-bytestring', 'ip6-dotint', "
                     "'no-ip6-dotint' is not allowed when IPv6 is disabled.",
                 )
+
+        if connection["match"]:
+            if connection["match"]["path"]:
+                if mode == self.VALIDATE_ONE_MODE_INITSCRIPTS:
+                    # initscripts does not support match.path, so raise errors when network
+                    # provider is initscripts
```

Comment:
This comment can be removed IMHO, the code is only describing the following code but not adding additional information.

Reply:
Removed the comment.